src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumeSQLRepository.cs

```diff
@@ -119,8 +124,9 @@ public Resume Update(Resume updateObject)
             throw new NotImplementedException();
         }
 
-        public IEnumerable<Skill> UpdateResumeSkill(int idResume, Skill[] skills)
+        public Resume UpdateResumeSkill(int idResume, Skill[] skills)
         {
+            var result = new Resume { Skills = skills.ToList() };
             var storeProcedure = "Resume_Skill_Update";
             var createTempTable = "CREATE TABLE #SkillTemp(Id Varchar(20), Name Varchar(50))";
```

Comment:
question, is this temp table deleted after it's being used?

Reply:
Yes, after is deleted